test/integration/annotation/annotated_resource_test.go

```diff
@@ -37,6 +37,7 @@ func TestMain(m *testing.M) {
 // table that starts verifying initial state and then applies modifications
 // to validate if Skupper is reacting as expected.
 func TestAnnotatedResources(t *testing.T) {
+	t.Skip("Disabled for now, it is too flaky probably due to a resources change in the circle ci")
```

Comment:
@nicob87 if we disable this, I would say it might be better to create an issue first.
Just to make sure we are not hiding an eventual bug.

Reply:
will try to reenable before merging (retrying if necessary)